lib/src/postgrest_error.dart

```diff
@@ -7,16 +7,16 @@ class PostgrestError {
     this.code,
   });
 
-  String message;
-  String details;
-  String hint;
-  String code;
+  String? message;
```

Comment:
let make `message` field not nullable. The rest can be nullable.

Reply:
Okay, I had to create messages in postgrest_builder.dart lines 128-137, because of this change. Let me know if the messages look good.